### DISCUSSION TITLE
Starting Different Projects but Deploy on same endpoint

### DISCUSSION THREAD

**Discussion by 95y410nq** [brefphp/bref#929] [General]
hello, just notice that when we deploy our project with bref into aws lambda.. even with different new project it will always deploy into same endpoint.. is there any configuration we are missing to make it deploy into different endpoint with diffrent project?

thanks before..

**Comment by t-richard** [comment]
Maybe it's deploying to the same stack ?

AFAIK Serverless framework will update the stack based on it's name which is the concat of the service name (`service: something` in serverless.yml) and the stage (taken either from `provider.stage` or the `--stage` option when deploying or `dev` by default).

So maybe you're just using the same service name and stage in all your projects ?

You could also check the number of Cloudformation stacks you have in the Cloudformation console (there should be one per project+stage combination).

**Reply by 95y410nq** [reply]
ah.. i see... we always leave the service name to default .. that's why then.. guess we miss the configuration in the documentation...  
thank youuuu...

**Reply by t-richard** [reply]
Glad it helped !

Not sure if that is something worth documenting in Bref because it's not specific to Bref but is a behaviour of Serverless Framework (to be honest, it's not super clear in their documentation either :sweat_smile:).

The only thing that could be done is change the default value to something that would emphasize more on the fact it should be changed.
